cmd/xmidt-agent/config.go

```diff
@@ -26,6 +28,41 @@ type Config struct {
 	Storage          Storage
 }
 
+type Client struct {
```

Comment:
Rename `Client` to be something like `Websocket` that way we can support this and `QUIC` in the future.